src/main/java/core/basesyntax/service/impl/BalanceTransactionHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+
+public class BalanceTransactionHandler implements TransactionHandler {
+    @Override
+    public void handleTransaction(FruitTransaction fruitTransaction, FruitService fruitService) {
+        if (Storage.remnantsOfGoods.containsKey(fruitTransaction.getFruit())) {
+            fruitService.addFruit(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
+        } else {
+            fruitService.setFruit(fruitTransaction.getFruit(),
+                    fruitTransaction.getQuantity());
+        }

```

Comment:
```suggestion
            fruitService.setFruit(fruitTransaction.getFruit(),
                    fruitTransaction.getQuantity());
```
We should just set for Balance operation